File: system-test/dotnet/SystemTests/Latest/AcceptanceTests/Base/BaseAcceptanceTest.cs
using Dsl.Port;
using Dsl.Core.Shared;
using SystemTests.Latest.Base;

namespace SystemTests.Latest.AcceptanceTests.Base;

public abstract class BaseAcceptanceTest : BaseScenarioDslTest
{
    protected override ExternalSystemMode? GetFixedExternalSystemMode()
    {
        return ExternalSystemMode.Stub;
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/latest/acceptance/base/BaseAcceptanceTest.java
package com.mycompany.myshop.systemtest.latest.acceptance.base;

import com.mycompany.myshop.systemtest.latest.base.BaseScenarioDslTest;
import com.mycompany.myshop.systemtest.configuration.ExternalSystemMode;

public abstract class BaseAcceptanceTest extends BaseScenarioDslTest {
    @Override
    protected ExternalSystemMode getFixedExternalSystemMode() {
        return ExternalSystemMode.STUB;
    }
}


